### DISCUSSION TITLE
AppImage

### DISCUSSION THREAD

**Discussion by lucasmz1** [strawberrymusicplayer/strawberry#943] [Ideas]
can you provide appimage version from the newest version of strawberry ?

**Comment by jonaski** [comment]
I stopped creating AppImages, it's a lot of work to maintain and support, it only benefits a few users. Most distros have Strawberry in their repository anyway.

**Reply by jonaski** [reply]
See https://forum.strawberrymusicplayer.org/topic/1094/appimages-for-linux
